Arduino-Bling/RobotBling/RobotBling.ino

```diff
@@ -0,0 +1,209 @@
+/*
+ * Spartronics Arduino Robot Bling Integration
+ *
+ * Arduino code for animation support:
+ * - uses rgbColor struct to break color into its components for easy reference
+ * - listens on serial port for new animation commands
+ * - ALL animations require call to `delay()` tp listen for interruptions
+ *
+ * Important:
+ * - Expected commands: ascii printable characters 0 and above
+ * - For development, start Arduino's serial port to test commands and animations
+ *
+ */
+
+/*
+ * Used by the magic _delay() function for running animations
+ */
+#include <setjmp.h>
+
+/*
+ * Setup Adafruit NeoMatrix
+ * - Important: validate PIN connection and number of LEDs
+ * - Note: this NeoMatrix does not have 'w' component
+ */
+#include <Adafruit_NeoPixel.h>
+
+// Configuration for our LED strip and Arduino
+// TODO: Update PIN and NUM_LEDS for development and later robot deployment
+#define NUM_LEDS 60
+#define PIN 4
+
+// allocate our pixel memory, set interface to match our hardware
```

Comment:
Consistency in comments: Upper-case the first word always (or never). I strongly prefer always. Indicates the beginning of a sentence, and not that a line above has been accidentally deleted.